onnxruntime/core/session/inference_session.cc

```diff
@@ -52,6 +54,77 @@
 
 using namespace ONNX_NAMESPACE;
 
+ONNXTensorElementDataType MLDataTypeToOnnxRuntimeTensorElementDataType(const onnxruntime::DataTypeImpl* cpp_type);
+const onnxruntime::DataTypeImpl* TensorElementDataTypeToMLDataType(ONNXTensorElementDataType type);
+
+namespace onnxruntime {
+const char* ElementTypeToString(MLDataType type) {
```

Comment:
>ElementTypeToString [](start = 12, length = 19)

Doesn't this exist already? I thought there was something in onnx that did this. If not, this file doesn't feel like the right place for it.

Reply:
Yeah, I was hoping someone would know, but as you saw the closest was in the test code which was less trivial to hook up.

Reply:
Can we add this in utils?